packages/nx-plugin/src/generators/package/vue/app/index.ts

```diff
@@ -1,39 +1,29 @@
 import * as path from 'path';
 import {
   generateFiles,
-  getWorkspaceLayout,
   joinPathFragments,
-  names,
   updateJson,
   type Tree,
 } from '@nrwl/devkit';
 import generateWebApp from '../../web/app';
 import generateCypressCt from '../../cypress/ct';
 import { installDependencies, installDevDependencies } from '@/utils/dependencies';
+import getPackageNames from '@/utils/names';
 
 interface Schema {
   name: string;
   description: string;
-  directory: string;
+  directory?: string;
```

Comment:
Why are directories optional in some of the generators?

Reply:
By default, the directory will be automatically determined based on the name.